Cuarta versión de la tarea 2

H-Cote opened the pull request:
Se corrigen errores ortográficos y se describe la convergencia de las funciones en términos de sus pendientes.

H-Cote committed:
Cuarta versión de la tarea 2

Se corrigen errores ortográficos y se describe la convergencia de las funciones en términos de sus pendientes.

lbenet commented:
Sobre la primer pregunta, la frase "El comportamiento general es un aumento en el error conforme $h$ disminuye." no es del todo cierta, ya que el error disminuye sólo si `h` es suficientemente pequeña. Lo de "en general" es incorrecto. Lo mismo se aplica en la ´¨ltima frase de la pregunta 2. Por cierto, no has corregido el error; escribiste "dismunuye"...

Respecto a lo que escribes en la pregunta 4, "En cuanto a la derivada compleja el error queda acotado después de llegar al punto donde la derivada es más exacta", es cierto; sin embargo, eso se debe a los números de punto flotante. La pendiente no cambia porque el error es tan pequeño que los cambios son esencialmente cero.

Creo que vale la pena que quites las ambigüedades, y que trates de ser más cuantitativo respecto a las pendientes (aunque hagas estimaciones), ya que justo eso es lo que determina la convergencia.

H-Cote committed:
Quinta versión de la tarea 2

Se corrigen errores ortográficos y se modifica la respuesta a la primera pregunta del ejercicio 4.

lbenet commented:
Acepto la tarea tal cual está; hay varios errores de dedo aún, pero lo demás está en general bien.
